server/ladders-remote.ts

```diff
@@ -28,7 +26,7 @@ export class LadderStore {
 	 * client.
 	 */
 	async getTop(prefix?: string): Promise<[string, string] | null> {
-		return null;
+		return new Promise(resolve => resolve(null));
```

Comment:
???

Reply:
First of all, `Promise.resolve` exists; second of all, it's completely unnecessary because `async` functions already wrap their return value in a Promise.

If eslint can't figure this out, just turn off the eslint check for it.